### ISSUE TITLE
[PAGE ISSUE]: 'An introduction to integration testing'

### ISSUE THREAD

**Issue by zpsean** [flutter/website#6942]
### Page URL

https://docs.flutter.dev/cookbook/testing/integration/introduction/

### Page source

https://github.com/flutter/website/tree/main/src/cookbook/testing/integration/introduction.md

### Describe the problem

 After executing the command:

```
flutter drive \
  --driver=test_driver/integration_test.dart \
  --target=integration_test/app_test.dart \
  -d chrome
```
One Chrome is opened. Be noticed that it is not labelled with 'chrome is controlled by automated test software'.

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/14213337/159384051-84080cd6-e34c-4ff6-ad71-b074d7dadd36.png">

Then try to execute the command plus arg `--no-headless`:

```
flutter drive \
  --driver=test_driver/integration_test.dart \
  --target=integration_test/app_test.dart \
  -d chrome \
  --no-headless 
```

Two Chrome instances are launched:
- One [backend one] is launched as device [this is NOT intend for test cases execution]
- The other [frontend one] is launched via webdriver (chromedriver this case) [this is intend for test cases execution, labelled with 'chrome is controlled by automated test software']

[**Note**: Mostly, the two Chromes are overlaid, please try to drag it away (up or down), then back one would be visible.]

<img width="1641" alt="image" src="https://user-images.githubusercontent.com/14213337/159163845-67e37dfb-b21d-474d-96ee-7d4c711fa140.png">

### Expected fix

Issue code raised: https://github.com/flutter/flutter/issues/100447
[Web][Integration Test] Two Chrome instances launched, and the unnecessary Chrome instance breaks the foundation of the integration test [Edited]

Before issue 100447 is fixed. Please kindly add above info in this page to clear the context to avoid the misleading to users.

**Comment by danagbemava-nc** [comment]
Hi @zpsean, I'm not sure I fully understand what your request is. 

Are you saying the page should recommend using `--no-headless` ?

**Comment by zpsean** [comment]
@danagbemava-nc in summary, as stated in: https://github.com/flutter/flutter/issues/100447
the function of flutter drive for integration_test has issues. that is, if use command below, the expected results need to be addressed as no Chrome is launched. However, currently, it has one Chrome launched wrongly, which will mislead the user. I suggest here need comments to clarify this in the page.
```
flutter drive \
  --driver=test_driver/integration_test.dart \
  --target=integration_test/app_test.dart \
  -d chrome
```

**Comment by zpsean** [comment]
Yes, maybe most of the users would try to use real browser (like Chrome) but not `-d web-server`, as the command itself also gives such suggestion.

**The web-server device requires the Dart Debug Chrome extension for debugging. Consider using the Chrome or Edge devices for an improved development workflow.**

```
%flutter drive --driver=test_driver/IT_driver_test.dart --target=integration_test/IT_test.dart -d web-server  
            
Running "flutter pub get" in myapp...                               3.1s
Launching integration_test/IT_test.dart on Web Server in debug mode...
Waiting for connection from debug service on Web Server...         27.3s
integration_test/IT_test.dart is being served at http://localhost:62423
The web-server device requires the Dart Debug Chrome extension for debugging. Consider using the Chrome or Edge devices for an improved development workflow.
All tests passed.
Application finished.

```

**Comment by danagbemava-nc** [comment]
Thanks for feedback

**Comment by atsansone** [comment]
@yjbanov , I know that the original issue(s) were closed in flutter/flutter (https://github.com/flutter/flutter/issues/100447, https://github.com/flutter/flutter/issues/67090), but I would like to provide better guidance in the docs. Do you have a recommendation I can include to avoid the double browser effect? I understand why it would happen, but I agree that this could make the tests wonky.
